### PR TITLE
add amp validation task

### PR DESCRIPTION
**Pull request by sebastianbenz**
We'll probably need more lightweight tasks as well...

### CODE REVIEW

File: gulpfile.js/validate.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+const {src} = require('gulp');
+const {join} = require('path');
+const gulpAmpValidator = require('gulp-amphtml-validator');
+
+/**
+ * Validates all pages build into /platform/pages.
+ */
+function validate() {
+  return src(join(__dirname, '../platform/pages/**/*.amp.html'))
```
Comment (matthiasrohmer) [inline gulpfile.js/validate.js]: Could use `@lib/utils/project.paths.GROW_BUILD_DEST` to be prepared when the build destination sometime gets move to `dist`.

File: examples/source/1.components/amp-web-push/static/sw.js [hunk 2]
```diff
@@ -18,17 +18,9 @@ importScripts('/sw-toolbox/sw-toolbox.js');
 importScripts('https://cdn.rawgit.com/jakearchibald/idb/97e4e878/lib/idb.js');
```
Comment (matthiasrohmer) [inline examples/source/1.components/amp-web-push/static/sw.js]: Was linting/fixing this file on purpose?

Comment (sebastianbenz) [reply examples/source/1.components/amp-web-push/static/sw.js]: yes - is there a reason why it shouldn't be?

Comment (matthiasrohmer) [reply examples/source/1.components/amp-web-push/static/sw.js]: Not really 😉was thinking about browser compatibility but every browser that does support SWs should also support modern syntax.

File: gulpfile.js/validate.js (hunk 1 above)
Comment (sebastianbenz) [reply gulpfile.js/validate.js]: good point - done

**Pull request merged**